tutorial.md

```diff
@@ -1,3 +1,1555 @@
-# How to use my script
+# Tutorial: how to use `vendor_data.py`
 
-In this notebook you will learn how to use the functions and classes defined in `script.py`.
+## Using `vendor_data.py` as a module
+
+This interactive notebook shows how to use the class and the different functions in the script `vendor_data.py`.
+
+To use the script in this code, we can import the script as a module or run it directly.
+We'll begin with importing the code as module:
+
+
+```python
+import sys
+sys.path.append ('vendor_data.py') #this is the relative path to the script. If the script is not directly next to the notebook, it could be better to copy the full path of the notebook.
+import vendor_data as vd # 'as vd' is not necessary but makes it shorter.
+```
+
+### Docstrings
+The script is completely documented with docstrings in order to ensure that the script is correclty used.
+You can read these docstrings by calling one of the following functions:
```

Comment:
Do not leave this output, it is too much automatic text for a tutorial.
You can show the code without running it by adding it inside the Markdown cell, with `` ```python `` right before and `` ``` `` afterwards.
For the tutorial, think of what you would like to find as a user that doesn't know about the module and is just starting. If anything, this kind of "here you have more information" could go _at the end_, but starting a tutorial with this can be a bit offputting for a user.